documentation/modules/security/con-securing-client-authorization.adoc

```diff
@@ -33,18 +33,3 @@ You can disable it using the `STRIMZI_ACLS_ADMIN_API_SUPPORTED` environment vari
 If no authorization is specified, the User Operator does not provision any access rights for the user.
 Whether such a `KafkaUser` can still access resources depends on the authorizer being used.
 For example, for `simple` authorization, this is determined by the `allow.everyone.if.no.acl.found` configuration in the Kafka cluster.
-
-== ACL rules
-
-`simple` authorization uses ACL rules to manage access to Kafka brokers.
-
-ACL rules grant access rights to the user, which you specify in the `acls` property.
-
-For more information about the `AclRule` object, see the link:{BookURLConfiguring}#type-AclRule-reference[`AclRule` schema reference^].
-
-== Super user access to Kafka brokers
-
-If a user is added to a list of super users in a Kafka broker configuration,
-the user is allowed unlimited access to the cluster regardless of any authorization constraints defined in ACLs in `KafkaUser`.
-
-For more information on configuring super user access to brokers, see xref:con-securing-kafka-authorization-{context}[Kafka authorization].
```

Comment:
Do we have this covered somewhere else?

Reply:
Yes. It's covered here: [15.2.1. Designating super users](https://strimzi.io/docs/operators/in-development/deploying#designating_super_users)